spec/factories/responses.rb

```diff
@@ -35,6 +35,7 @@
       }
     end
     limit      { "" }
+    ask_limit_on_launch true
```

Comment:
This doesn't apply to everything, can you follow the trait patterns below?

Reply:
done